README.md

```diff
@@ -122,6 +122,7 @@ Distributed Locks
 - [dogapi](#dogapi)
 - [dropbox-sdk](#dropbox-sdk)
 - [droplet_kit](#droplet_kit)
+- [faktory](#faktory)
```

Comment:
Would be good to match the gem name (`faktory_worker_ruby`). Also, should probably go in the "Other" section instead of "3rd party services" since it's self-hosted.